macros/DOMxRef.ejs

```diff
@@ -21,13 +21,17 @@ let str = $1 || $0;
 let rtlLocales = ['ar', 'he', 'fa'];
 let localStrings = string.deserialize(await template("L10n:Common"));
 
-api = api.replace(' ', '_').replace('()', '').replace('.prototype.', '.').replace(/\./g, '/');
+api = api.replace(/ /g, '_')
+    .replace(/\(\)$|\(\)(?=\.)/g, '')
```

Comment:
Sorry to be a stickler about this, but since we don't have tests for this macro I don't want to risk replacing a very simple string replace with a complicated regexp like this.

I'll accept an unchanged `.replace('()', '')` here, or a simple regexp `.replace(/\(\)/g, '')`

If replacing all '()' that is not at the end of a line or before a period is a bug that need to be fixed here you'll have to convince me of that.

Your code is arguably the most correct way to do this, and it looks like it works. But it also seems more complicated than necessary, and we don't have tests, and I don't want to risk changing current behavior.

If someone has written a macro {{domxref('()foo')}}, this patch would change its output. That probably isn't the case, but why risk it?

Reply:
```suggestion
    .replace(/\(\)/g, '')
```